client/src/App.jsx

```diff
@@ -17,27 +17,29 @@ class App extends React.Component {
     }
   }
 
-  componentDidMount() {
-    axios.get('/products')
-    .then(response => {
-      let productList = response.data;
-      this.setState({
-        products: productList,
-        productId: productList[0].id
-      });
-    })
-    .catch(err => {
-      console.log(err);
-    });
-  }
+  // Commented this one out so I don't
+  // make network calls for now.
+  // Just using sampleData for now.
+  // componentDidMount() {
+  //   utils.getProducts()
+  //   .then(productList => {
+  //     this.setState({
+  //       products: productList,
+  //       productId: productList[0].id
+  //     });
+  //   })
+  //   .catch(err => {
+  //     console.log(err);
+  //   });
+  // }
```

Comment:
This is fine, but this code doesn't affect your ability to use sample data. You can just import the sampleData file directly to your component and ignore the prop that would have been passed in on new line 42.